odkx-src/sync-endpoint.rst

```diff
@@ -132,3 +143,129 @@ Assigning users to groups
+    
+    2. Copy the bootstrap.ldif file from the OpenLDAP directory to the new directory .In the docker file Add the image of the ldap directory to be used and add 
+       the "COPY" command to copy the bootstrap.ldif file to the right path in the container .
+    
+    3. Run the following command to build the docker image :
+    
+     .. code-block:: console
+  
+       $ docker build -t odk/[LDAP_DIRECTORY_NAME] [ Folder conatining the docker file ]
+       
+    4. Replace the ldap-service image from docker-compose.yml with odk/[LDAP_DIRECTORY_NAME].
+    
+    5. Modify config/sync-endpoint/security.properties to fill in the Settings for LDAP server.
+       Set security.server.ldapUrl in security.properties to the new server url . The name of the service in swarm would be same ( ldap-service ) . So just change 
+       the port number .
+    
+.. _sync-endpoint-ldap-ui:
+
+Using a Different LDAP UI
+""""""""""""""""""""""""""""""""""""""""""""""
+    
+    If you want to use a UI outside the docker swarm in your local machine Modify the docker-compose.yml file in sync-endpoint-default-setup directory . Add ports 
+    mapping to the ldap service to expose the port 389 of ldap service to a port in your local host . If you wish to access 
+    the ldap protocol over TLS/SSL expose the port 636. Connect the UI application to this port on localhost.
+    
+    The ldap service of the the docker compose should be like this after adding port mapping .
+    
+    .. code-block:: console
+        
+      ldap-service:
+       image: odk/openldap
+       deploy:
+        replicas: 1
+       networks:
+        - ldap-network
+       ports:
+        - "YOUR_LOCAL_HOST_PORT:389"    // 389 is the default port of openLDAP 
+       volumes:
+        - ldap-vol:/var/lib/ldap
+        - ldap-slapd.d-vol:/etc/ldap/slapd.d
+       env_file:
+        - ldap.env 
+    
+    
+    For running the UI application in the docker swarm create a folder in the sync-endpoint-default-setup directory and create a docker file inside it .
+    Copy the templates folder from the phpLDAPadmin directory to the new directory . In the docker file ,add the image of the UI application to be used and the 
+    "COPY" command to copy the templates folder to the right path inside the container .
+    
+    To build the docker image run the command in the sync-endpoint-default-setup-directory with tag odk/[YOUR_UI_APPLICATION_NAME] :
+    
+     .. code-block:: console
+  
+       $ docker build -t odk/[YOUR_UI_APPLICATION_NAME] [ Folder conatining the docker file ]
+       
+    Edit the docker-compose.yml file . Replace the image of phpLDAPadmin service with odk/[YOUR_UI_APPLICATION_NAME] . 
+    
+    
+.. _sync-endpoint-dhis2:    
+
+Managing Identity through DHIS2
+"""""""""""""""""""""""""""""""""
+    Modify config/sync-endpoint/security.properties to fill in the Settings for DHIS2 Authentication section
```

Comment:
The formatting of this paragraph looks a bit awkward. Was it supposed to be an ordered list?